Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -40,9 +42,33 @@ protocol TextStorageAttachmentsDelegate {
     /// - Parameters:
     ///   - textView: The textView where the attachment was removed.
     ///   - attachmentID: The attachment identifier of the media removed.
-    func storage(_ storage: TextStorage, deletedAttachmentWithID attachmentID: String);
+    ///
+    func storage(_ storage: TextStorage, deletedAttachmentWithID attachmentID: String)
+
+    /// Provides the Bounds required to represent a given attachment, within a specified line fragment.
+    ///
+    /// - Parameters:
+    ///     - storage: The storage that is requesting the bounds.
+    ///     - attachment: CommentAttachment about to be rendered.
+    ///     - lineFragment: Line Fragment in which the glyph would be rendered.
+    ///
+    /// - Returns: Rect specifying the Bounds for the comment attachment
+    ///
+    func storage(_ storage: TextStorage, boundsForComment attachment: CommentAttachment, with lineFragment: CGRect) -> CGRect
```

Comment:
This protocol is starting to feel a bit mixed up and over complex, between images and comments, maybe two different protocols?